tests/data/experiments/lr-eval-cfg/output/lr_eval_cfg_eval_by_QUESTION.csv

```diff
@@ -1,7 +1,7 @@
-,N,R2.scale_trim,RMSE.scale_trim,SMD.scale_trim,SMD.scale_trim_round,adj_agr.scale_trim_round,corr.scale_trim,exact_agr.scale_trim_round,h_mean,h_sd,kappa.scale_trim_round,sys_mean.scale_trim,sys_mean.scale_trim_round,sys_sd.scale_trim,sys_sd.scale_trim_round,wtkappa.scale_trim_round
```

Comment:
This table should also now have wtkappa.raw_trim - I believe it's the same list of short metrics.

Reply:
Just to confirm - you mean `scale_trim`, correct? We're not changing whether to use `raw` or `scale` here; we're only changing whether to use the rounded or unrounded scores to calculate QWK, right?

Reply:
Yes, that's correct.